### DISCUSSION TITLE
Suggestion: Rethink pop-up for new Pluto version

### DISCUSSION THREAD

**Discussion by mkschleg** [fonsp/Pluto.jl#2101] [Pluto development]
I think it is a great idea to include some way to know there is a new Pluto version, but it is a bit annoying when there is a pop-up every time I go to the main page of my running server. 

Maybe there could be a pop-up once, and some indicator on the main page that there is an update available?

For more info on my use case. I can have Pluto server running for several days/weeks w/o closing so it is very possible a new update will come out in-between restarts. Restarting for a minor update can be really painful (i.e. need to re-open lots of notebooks and chug through them).

**Comment by rikhuijzer** [comment]
I’m impressed that you keep your notebooks open for weeks. That’s also a way to solve the time to first X (TTFX) problem

**Reply by mkschleg** [reply]
I think it is rather the server stays open, not any particular notebook. I tend to restart when I feel a need to update my system image for a particular project (which can be really infrequent).

**Comment by fonsp** [comment]
Since about a year ago, we only show the popup for very major updates, for which we manually enabled the popup.

**Reply by mkschleg** [reply]
What constitutes "very major updates"? I'm on v0.19.3 and had been getting pop-ups since v0.19.4 was released.
